### ISSUE TITLE
Дополнение для Firefox на Android

### ISSUE THREAD

**Issue by ObhodBlokirovok** [anticensority/runet-censorship-bypass#86]
Здравствуйте!

1. Есть версия расширения [Обход блокировок Рунета](https://addons.mozilla.org/ru/firefox/addon/обход-блокировок-рунета/) на десктопный Firefox. У того же Firefox есть Android-версия приложения с [поддержкой дополнений](https://addons.mozilla.org/ru/android/). Возможно ли реализовать поддержку расширения [Обход блокировок Рунета](https://addons.mozilla.org/ru/firefox/addon/обход-блокировок-рунета/) для Firefox на Android, пожалуйста? Было бы очень удобно.

2. Расширение вроде как в Яндекс Браузере для Android завелось, но данный браузер мне не подходит по определённым причинам. Есть ли другие браузеры на Android с поддержкой данного расширения? В идеале, конечно, поддержку в Firefox на Android добавить.

**Comment by ilyaigpetrov** [comment]
Я поставил галочку поддержки Android-версии при последней публикации
расширения в магазин Мозиллы, но совсем не тестировал.
Вот здесь один пользователь поделился своей историей про попытку постаить
расширение на FireFox под Андроид:
https://addons.mozilla.org/en-US/firefox/addon/обход-блокировок-рунета/reviews/1630611/
.

**Comment by ilyaigpetrov** [comment]
Вы также можете настроить наш бесплатный vpn под Андроидом, см. инструкции
на https://antizapret.prostovpn.org.

**Comment by ObhodBlokirovok** [comment]
> Вы также можете настроить наш бесплатный vpn под Андроидом, см. инструкции

Не хочется устанавливать VPN в систему. Просто нужен отдельный браузер для доступа к заблокированным сайтам. Браузер + расширение для меня самое удобное решение, а под Fifrefox уже есть пара нужных мне дополнений, только расширения **Обход блокировок Рунета** не хватает для полного счастья...

> Я поставил галочку поддержки Android-версии при последней публикации расширения в магазин Мозиллы, но совсем не тестировал.

А как установить и потестировать дополнение на Fifrefox для Android, если не секрет? К слову, на Яндекс Браузере на Android из магазина Chrome расширение установилось и вполне себе работало.

**Comment by ghost** [comment]
> Просто нужен отдельный браузер для доступа к заблокированным сайтам

Then you can use any Firefox-based browser with **about:config** support, such as Iceraven.
Just go to the **about:config** page, change the **network.proxy.type** value to **2** and the **network.proxy.autoconfig_url** value to **https://antizapret.prostovpn.org/proxy.pac**

**Comment by alexsm88** [comment]
Обнаружена  ошибка.
1. ...при запуске Firefox 91.0 в Windows 7, расширение anticensority остаётся не активным.
2. ...после выключения-включения (перезапуска) расширение начинает работать.

**Comment by ilyaigpetrov** [comment]
@alexsm88, это на Android?
Если ошибка не связяна с Андроидом, то, пожалуйста, создайте новый билет (new issue).
Прошу не спамить.

**Comment by InvisibleRain** [comment]
При запуске на андроиде в Firefox Nightly 97 выдаёт ошибку в настройках аддона:
`proxy.settings is not supported on android.`

Получается тут только ждать когда Mozilla добавит поддержку?

**Comment by ilyaigpetrov** [comment]
Да. Вы можете ускорить процесс реализации данной функции, проголосовав за
неё здесь: https://bugzilla.mozilla.org/show_bug.cgi?id=1725981 (details ->
vote).

**Comment by ilyaigpetrov** [comment]
Только что было закрыто обсуждение https://github.com/anticensority/runet-censorship-bypass/discussions/261 как дубликат -- там, помимо прочего, был [отзыв](https://github.com/anticensority/runet-censorship-bypass/discussions/261#discussioncomment-10139649) к версии о 0.1.66.
